scripts/lint-packages.sh

```diff
@@ -98,6 +144,15 @@ lint_package() {
 	fi
 	echo "PASS"
 
+	echo -n "Indentation check: "
+	for script in "$package_script" $(dirname "$package_script")/*.subpackage.sh; do
+		test ! -f "$script" && continue
+		check_indentation "$script" || return 1
+	done
+	echo "PASS"
+
+	echo
```

Comment:
Extra echo needed?

Reply:
Not needed, just looks better.
I can make it a `echo -e "PASS\n"` above instead.

Reply:
Same thing is done at the trailing whitespace check actually.

Reply:
I'll group these first five checks then.
![image](https://github.com/termux/termux-packages/assets/43716232/7333183d-f2d8-4d04-b7dc-bf7ade3c1d18)

Reply:
![image](https://github.com/termux/termux-packages/assets/43716232/3bae853d-1453-4c0e-80fe-3e8dfdd36ad4)
There we go.